biom/table.py

```diff
@@ -522,6 +521,61 @@ def matrix_data(self):
         """The sparse matrix object"""
         return self._data
 
+    def to_pandas(self):
+        """ Converts biom table into 3 pandas dataframes.
+
+        Returns
+        ----------
+        counts : pd.SparseDataFrame
+            Pandas representation of table values (i.e. counts)
+        sample_metadata : pd.DataFrame
+            Pandas representation of the sample metadata
+        observation_metadata : pd.DataFrame
+            Pandas representation of the observation metadata
+
+        Note
+        ----
+        This assumes that the keys across the metadata fields are consistent.
+        For example, if the first observation has taxonomy, it is assumed that
+        all of the observations also have taxonomy.
+        """
+        # Double check to see if pandas is installed properly
+        try:
+            import pandas as pd
+        except:
+            ImportError("pandas installation not found")
+
+        # Extract count data
+        m = self.matrix_data
+        data = [pd.SparseSeries(np.squeeze(row.toarray()))
```

Comment:
would [`from_coo()`](http://pandas.pydata.org/pandas-docs/stable/generated/pandas.SparseSeries.from_coo.html#pandas.SparseSeries.from_coo) be more appropriate?